cfgov/v1/util/filterable_list.py

```diff
@@ -22,10 +22,17 @@ def get_context(self, request, *args, **kwargs):
         context['filter_data'] = self.process_form(request, form)
         return context
 
-    def base_query(self):
-        return AbstractFilterPage.objects.live().filter(
+    def base_query(self, site):
+        return AbstractFilterPage.objects.live().in_site(site).filter(
             CFGOVPage.objects.child_of_q(self))
 
+    def process_form(self, request, form):
```

Comment:
Is this a copy/paste error? This code is duplicated from below.

Reply:
Sure is - I rebased. Thanks for catching that.